src/player.js

```diff
@@ -1248,6 +1249,21 @@ export default class Player extends FakeEventTarget {
     return false;
   }
 
+  _applyTextTrackConfig(): void {
+    if (Utils.Object.getPropertyPath(this._config, 'text.forceCenterCaptions')) {
+      this.setTextDisplaySettings({position: 'auto', align: 'center', size: '100'});
+    } else if (Utils.Object.hasPropertyPath(this._config, 'text.textTrackDisplaySetting')) {
+      this.setTextDisplaySettings(this._config.text.textTrackDisplaySetting);
+    }
+    try {
+      if (Utils.Object.hasPropertyPath(this._config, 'text.textStyle')) {
+        this.textStyle = this._config.text.textStyle;
```

Comment:
need to use the `textStyle` setter as it handles also native text track and in general we should reuse code.
It means you need to create a `TextStyle` instance.
As this is coming from a JSON config we may add to `TextStyle` a static `fromJson` that gets json and returns a populated instance.

Reply:
I used the textStyle setter, as part from https://github.com/kaltura/playkit-js/commit/f06aa5e929f8ab947e10ee3a466a3cc90cd0f62c we can pass instance by config

Reply:
You are right, you use it.
Issue is that when coming from uiconf it won't be instance of TextStyle so we need to normalize it either in Kaltura player when getting this from uiconf or here.